HVZ.Persistence/Models/Organization.cs

```diff
@@ -51,19 +51,19 @@ public class Organization : IdEquatable<Organization>
     /// <summary>
     /// Description of the Organization defined by the owner.
     /// </summary>
-    public string Description { get; set; } = string.Empty;
+    public string Description { get; set; }
 
     /// <summary>
     /// Whether players must have a verified email address before joining a game
     /// </summary>
-    public bool RequireVerifiedEmailForPlayer { get; set; } = false;
+    public bool RequireVerifiedEmailForPlayer { get; set; }
 
     /// <summary>
     /// Whether players must have uploaded a profile picture before joining a game
     /// </summary>
-    public bool RequireProfilePictureForPlayer { get; set; } = false;
+    public bool RequireProfilePictureForPlayer { get; set; }
 
-    public Organization(string id, string name, string ownerid, HashSet<string> moderators, HashSet<string> administrators, HashSet<Game> games, string? activegameid, Instant createdat, string url)
+    public Organization(string id, string name, string ownerid, HashSet<string> moderators, HashSet<string> administrators, HashSet<Game> games, string? activegameid, Instant createdat, string url, string description = "", bool requireVerifiedEmail = false, bool requireProfilePicture = false)
```

Comment:
```suggestion
    public Organization(string id, string name, string ownerid, HashSet<string> moderators, HashSet<string> administrators, HashSet<Game> games, string? activegameid, Instant createdat, string url, string description = string.Empty, bool requireVerifiedEmail = false, bool requireProfilePicture = false)
```

Reply:
doing so causes an error [CS1736](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1736?f1url=%3FappId%3Droslyn%26k%3Dk(CS1736)) `string.Empty` is not a compile-time constant